charts/sequencer/files/cometbft/config/config.toml

```diff
@@ -297,15 +297,15 @@ broadcast = true
 wal_dir = ""
 
 # Maximum number of transactions in the mempool
-size = 5000
+size = 2000
 
 # Limit the total size of all txs in the mempool.
 # This only accounts for raw transactions (e.g. given 1MB transactions and
 # max_txs_bytes=5MB, mempool will only accept 5 transactions).
-max_txs_bytes = 1073741824
+max_txs_bytes = 100_000_000
```

Comment:
why the change here?

Reply:
Decreasing mempool size from 1GB to 100 MB, better default configs, same for other values.